client/my-sites/promote-post/components/campaign-item/index.tsx

```diff
@@ -173,8 +172,29 @@ export default function CampaignItem( { campaign }: Props ) {
 
 			<FoldableCard header={ header } hideSummary={ true } className="campaign-item__foldable-card">
 				{ campaignStatus === 'rejected' && moderation_reason && (
-					<Notice status="is-warning" showDismiss={ false }>
-						{ __( 'Your ad is rejected: ' ) } { moderation_reason }
+					<Notice isDismissible={ false } className="campaign-item__notice" status="warning">
+						<Gridicon className="campaign-item__notice-icon" icon="info-outline" />
+						{ translate(
+							'Your ad was not approved, please review our {{wpcomTos}}WordPress.com Terms{{/wpcomTos}} and {{advertisingTos}}Advertising Guidelines{{/advertisingTos}}.',
```

Comment:
Please lets replace "Advertising Guidelines" to "Advertising Policy"

searching the calypso repo, this is the only place this is mentioned :)

Reply:
changed